packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.js

```diff
@@ -99,6 +69,47 @@ const ShippingMethod = props => {
                 />
             </Fragment>
         );
+    } else {
+        // We're either initializing or editing.
+        const lowestCostShippingMethodSerializedValue = shippingMethods.length
+            ? shippingMethods[0].serializedValue
+            : '';
+        const lowestCostShippingMethod = {
+            shipping_method: lowestCostShippingMethodSerializedValue
+        };
+
+        const isContinueDisabled = pageIsUpdating;
+
+        const editingContents = (
+            <Form
+                className={classes.form}
+                initialValues={lowestCostShippingMethod}
+                onSubmit={handleSubmit}
+            >
+                <ShippingRadios shippingMethods={shippingMethods} />
+                <div className={classes.formButtons}>
+                    <Button
+                        priority="normal"
+                        type="submit"
+                        disabled={isContinueDisabled}
+                    >
+                        {'Continue to Payment Information'}
+                    </Button>
+                </div>
+            </Form>
+        );
+
+        const bodyContents =
+            displayState === displayStates.INITIALIZING
+                ? initializingContents
+                : editingContents;
```

Comment:
Recent feedback from Jimmy, very minor. We've already paid the cost of creating the `editingContent` element and this means we may never render it. Best to only create element we intend to render.